application/src/main/java/org/opentripplanner/model/impl/SubmodeMappingRow.java

```diff
@@ -0,0 +1,9 @@
+package org.opentripplanner.model.impl;
```

Comment:
Idk if this package is the right place for this or some of the other classes/records in this pr. However, perhaps someone else could point to what would be the ideal package structure for this.